src/test/resources/org/broadinstitute/barclay/help/expected/HelpDoclet/org_broadinstitute_barclay_help_TestArgumentContainer.html

```diff
@@ -538,7 +536,7 @@
 		</p>
 		<hr style="border-bottom: dotted 1px #C0C0C0;" />
 		<h3><a name="--testPlugin">--testPlugin </a>
-			 / <small>-</small>
+			 / <small>NA</small>
```

Comment:
If the fix for this is in a different PR, it will be nice to change the template to handle NAs properly and remove them from the output.